src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -277,6 +277,12 @@ const StreamActions = createReactClass({
     });
   },
 
+  onCreateIncident() {
```

Comment:
Is this a scenario where you mentioned using the `handle-` prefix previously?

Reply:
Yeah, there was a lot of existing code, but I can update it